packages/salesforcedx-lwc-language-server/package.json

```diff
@@ -0,0 +1,128 @@
+{
+  "name": "@salesforce/salesforcedx-lwc-language-server",
+  "displayName": "LWC Language Server",
+  "version": "64.16.1",
+  "description": "Language server for Lightning Web Components.",
+  "publisher": "salesforce",
+  "engines": {
+    "vscode": "^1.90.0"
+  },
+  "main": "out/src/indexer.js",
+  "typings": "out/src/indexer.d.ts",
+  "sideEffects": false,
+  "exports": {
```

Comment:
same question here...only export what's imported by something else (keeps the API smaller and reduces the chances for having to make changes in 2 packages)